src/thread_local_heap.h

```diff
@@ -172,7 +174,7 @@ class ThreadLocalHeap {
   }
 
   // semiansiheap ensures we never see size == 0
-  inline void *ATTRIBUTE_ALWAYS_INLINE malloc(size_t sz) {
+  inline void *ATTRIBUTE_ALWAYS_INLINE ATTRIBUTE_MALLOC malloc(size_t sz) {
```

Comment:
why not put `ATTRIBUTE_SIZE(1)` here, too?

Reply:
because it does not help at all in this case, this attribute are hint for compiler to help determine where the size source(s) is gonna be but here no doubt possible.